ipython/estimate_solvation_thermo_and_search_available_solvents.ipynb

```diff
@@ -0,0 +1,236 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Estimate Solvation Thermo\n",
+    "\n",
+    "This script estimates solvation thermo (solute parameters, solvation free energy, and solvation enthalpy). It first checks whether the values can be found from the solute library, and if not, it uses the group additivity approach to estimate the solute parameters and solvation properties. Read documentation on solvation thermochemistry for detailed method (http://reactionmechanismgenerator.github.io/RMG-Py/users/rmg/liquids.html).\n",
+    "\n",
+    "You can also look up available solvents in RMG using a SMILES string with this script.\n",
+    "\n",
+    "Code written by Yunsie Chung"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os.path\n",
+    "from rmgpy import settings\n",
+    "from rmgpy.rmg.main import Species\n",
+    "from rmgpy.data.solvation import SolvationDatabase\n",
+    "\n",
+    "solvationDatabase = SolvationDatabase()\n",
+    "solvationDatabase.load(os.path.join(settings['database.directory'], 'solvation'))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Search for available solvents using a SMILES string"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def print_find_solvent_results(match_list):\n",
+    "    dGsolv_param_list = ['e_g', 's_g', 'a_g', 'b_g', 'l_g', 'c_g']\n",
+    "    dHsolv_param_list = ['e_h', 's_h', 'a_h', 'b_h', 'l_h', 'c_h']\n",
+    "    viscosity_param_list = ['A', 'B', 'C', 'D', 'E']\n",
+    "    if len(match_list) == 0:\n",
+    "        print('No matching solvent is found in the RMG solvent library')\n",
+    "    else:\n",
+    "        count = 1\n",
+    "        for label, entry in match_list:\n",
+    "            print(f'Matched Solvent {count}: {label}')\n",
+    "            solven_data = entry.data\n",
+    "            if solven_data.e_g is None:\n",
+    "                print('- Solvation free energy calculation is not availble for this solvent.')\n",
+    "            else:\n",
+    "                print('- Solvtaion free energy calculation is available. The relevant Abraham solvent parameters are:')\n",
+    "                print(f'\\te_g = {solven_data.e_g}, s_g = {solven_data.s_g}, a_g = {solven_data.a_g}, \\\n",
+    "b_g = {solven_data.b_g}, l_g = {solven_data.l_g}, c_g = {solven_data.c_g}')\n",
+    "            if solven_data.e_h is None:\n",
+    "                print('- Solvation enthalpy calculation is not availble for this solvent.')\n",
+    "            else:\n",
+    "                print('- Solvtaion enthalpy calculation is available. The relevant Mintz solvent parameters are:')\n",
```

Comment:
solvtation => solvation throughout

Reply:
It's fixed in this fixup commit 1b7f550.